setup.py

```diff
@@ -40,11 +40,10 @@
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
     ],
-    install_requires=[
-        'simplejson >= 2.0.9'],
     setup_requires=[
         'versiontools >= 1.3.1'],
     tests_require=[
+        'simplejson >= 2.0.9',
```

Comment:
This is somewhat tricky, it means that all the tests really depend on having simplejson available. What happens without simplejson? I assume things will still work but might produce different error messages. Is that the case?